src/main/java/seedu/duke/Warehouse.java

```diff
@@ -0,0 +1,112 @@
+package seedu.duke;
+
+import util.exceptions.LargeQuantityException;
+import util.exceptions.NullException;
+
+import java.util.ArrayList;
+
+public class Warehouse {
+    private int capacity;
+    private static ArrayList<Order> orderLists = new ArrayList<Order>();
+
+    public Warehouse(int capacity) {
+        this.capacity = capacity;
+    }
+
+    public void viewOrder(String orderId) {
+        try {
+            Integer idToBeViewed = Integer.parseInt(orderId);
+            for (Order order : orderLists) {
+                if (idToBeViewed.equals(order.getId())) {
+                    return;
+                }
+            }
+            System.out.println("Could not find item with given id!");
+        } catch (NumberFormatException e) {
+            System.out.println("Invalid format entered! Check format and try again!");
+        }
+    }
+
+    public void viewGood(String goodId) {
+        try {
+            Integer idToBeViewed = Integer.parseInt(goodId);
+            for (Order order : orderLists) {
+                for (Good good : order.getGoods()) {
```

Comment:
my current implementation of how order would hold the arraylist of goods, change as per required please @tuan0369